uefi/src/table/runtime.rs

```diff
@@ -245,6 +245,12 @@ impl RuntimeServices {
         }
     }
 
+    /// Deletes a UEFI variable.
+    ///
```

Comment:
nit: the trailing blank comment line can be removed.

Reply:
Fixed, thanks.